docs/application/dotnet/guides/app-management/app-manager.md

```diff
@@ -40,6 +40,12 @@ To get the application running context and its details, and to operate on the co
 
 1.  Get the context of the currently-running application by creating an instance of the [Tizen.Applications.ApplicationRunningContext](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.ApplicationRunningContext.html) class, with the ID of the application from which the context is being obtained as a parameter.
 
+    If you do not know your App ID, you can retrieve it as follows.
+
+    ```
+    string applicationId = ApplicationManager.GetAppId((Your PID);
```

Comment:
Would you please check `((Your PID)`?
Also, I can't see GetAppId from https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.ApplicationManager.html. 

Is this update really for master branch, the branch for tizen 5.0?

Reply:
This update is for 5.5.
http://suprem.sec.samsung.net/jira/browse/TCSACR-202

Reply:
Would you please rebase this PR? I switched to tizen_5.5_prepare branch, which is for tizen 5.5.

Reply:
I'll make new PR based on tizen_5.5_prepare